UI/BuildDesignatorWnd.cpp

```diff
@@ -323,6 +329,25 @@ namespace {
                 ClientUI::ShipDesignIcon(item.design_id), title, main_text);
         }
 
+        // production item is a stockpiling project
+        if (item.build_type == BT_STOCKPILE) {
+
+            // create title, description, production time and cost
+            const std::string& title = UserString("PROJECT_BT_STOCKPILE");
+            std::string main_text = UserString("PROJECT_BT_STOCKPILE_DESC");
+            float total_cost = 1.0;
+            int production_time = 1;
+
+            main_text += "\n\n" + UserString("PRODUCTION_WND_TOOLTIP_PROD_COST") + ": " + DoubleToString(total_cost, 3, false);
```

Comment:
prefer using formatter to in-C++ text layout for user-visible text

Reply:
Not sure what that means.